### DISCUSSION TITLE
POSTER OF WR-124

### DISCUSSION THREAD

**Discussion by RobeReyes** [will-henney/globule-seminario#35] [General]
Hi!

this is a first purpose to the poster. I will put the equations and images later, but for the moment I want to know if the content need corrections.

Also, I don't know if this is the correct format or I need to change. 

[POSTER WR-124.pdf](https://github.com/will-henney/globule-seminario/files/11741265/POSTER.WR-124.pdf)

**Comment by RobeReyes** [comment]
please, ignore the first paragraph of section 4.

**Comment by will-henney** [comment]
Hi Roberto. I think it is a good start. 

There are many ways to make a poster.  Your proposal looks quite formal and dense with text.  That is fine, if that is what you prefer, but you might want to think about a more visual approach.  I will send links to some posters I have done previously to give you an idea of what  I mean.

**Comment by will-henney** [comment]
https://www.dropbox.com/s/rjg438oor1do2w0/poster-luis-2014.pdf?dl=1

This is one on bow shocks

![poster-luis-2014-small](https://github.com/will-henney/globule-seminario/assets/161264/83d737fa-c698-4a24-8cfb-48656d0dd640)

**Comment by will-henney** [comment]
And this is another one, which is admittedly a bit extreme

https://www.dropbox.com/s/mmhb9sqtdui5yxz/wjh-townsville-poster.pdf?dl=1

![wjh-townsville-poster-small](https://github.com/will-henney/globule-seminario/assets/161264/96a6155b-0641-4628-b802-5ccf317a53be)

**Comment by will-henney** [comment]
So, as I said, you can decide what sort of style you want too use.  If you prefer to have it with more text and less figures, than that is fine

**Comment by will-henney** [comment]
Here is another example of a poster on the same nebula that Palmira did from a few years ago

![Palmira-WR-Poster](https://github.com/will-henney/globule-seminario/assets/161264/394ed4be-e6e6-45b6-97db-595c3e1906f9)

**Comment by RobeReyes** [comment]
What about this option? 
[Poster2.pdf](https://github.com/will-henney/globule-seminario/files/11776716/Poster2.pdf)

**Comment by RobeReyes** [comment]
And this is another option simplified 
[poster3.pdf](https://github.com/will-henney/globule-seminario/files/11777251/poster3.pdf)

**Reply by will-henney** [reply]
I think either of these would be great!  I have a few suggested edits for the the text, which I will send in following comments.

**Comment by will-henney** [comment]
> 1. Circumstellar Nebula M1-67
> In recent images from JWST we can see in great detail the presence of neutral globules around the WR-124. Once we located the globules in the JWST image, we obtained the brightness in the HST image.

Recent JWST images reveal the presence of numerous neutral globules around WR-124, many of which are associated with bright arcs of ionized gas emission, located between the globule and the star. We measured the H$\alpha$ emission properties of these arcs from archival HST images.

**Comment by will-henney** [comment]
> 2. Steady State Hydrodynamic Model
> In this case, we assume a steady state hydrodynamic model to compare the RAM pressure of the stellar wind with the internal pression of the globule. In the interaction between this two pressures, there is an equilibrium point, and from this, we can obtain others physical parameters.

We propose that these emission arcs are shocked shells resulting from a supersonic wind-wind interaction between (A) the stellar wind from the WR star, and (B) a transonic photoevaporation flow from the surface of the neutral globule that is induced by the ultraviolet radiation from the star. 

We employ a steady-state hydrodynamic model to calculate the equilibrium point where the ram pressure of the two winds is in balance. This is also equal to the thermal pressure of the shocked shell, which can be observationally estimated from the HST images.

**Comment by will-henney** [comment]
> 3. WR-124 Star

I suggest changing the heading to "Results"

> 4. Results

And changing this one to "Conclusions"

**Comment by will-henney** [comment]
> For this case we have that the RAM pressure of the stellar wind caused by the WR-124 is ...

I think it would be better to give a description of the figures here.  Also, you could label the figures with letters (A, B, C, ....) so you can refer to them. 

I do not have time to send more comments now, but I will come back later

**Comment by will-henney** [comment]
Figure captions (I suggest putting each one directly next to the corresponding figure)

![CleanShot 2023-06-18 at 14 27 07](https://github.com/will-henney/globule-seminario/assets/161264/deae2fd5-f42c-4082-8976-0c9e536555f1)

A: HST WFPC2 H$\alpha$ image of M1-67 nebula  (Grosdidier et al 1998). Image credit: ESA/NASA/Judy Schmidt

B: JWST NIRCAM image of M1-67. F444W, F335M, F210M, F150W, F090W. Image processing Judy Schmidt. 
(https://www.flickr.com/photos/geckzilla)

C. Zoom on group of neutral globules (pink) and interaction arcs (gray-white).

D. Cartoon of the interaction between two supersonic flows. The inner wind is slow and dense, while the outer wind is fast and low density.  Only the inner shell is expected to be visible. 

E. Examples of measuring the radius and surface brightness of the interaction arcs from the HST image.

**Comment by will-henney** [comment]
![CleanShot 2023-06-18 at 15 37 57](https://github.com/will-henney/globule-seminario/assets/161264/6e205437-37ec-4035-9a93-a01ca84db024)

F, G. Properties of the interaction shell as a function of radius r1 (normalized to globule radius r0): pre-shock Mach number M, pre-shock density rho1, shell density rho2, shell thickness h2, shell pressure P, radial emission measure E2, limb-brightened shell surface brightness S2, shell luminosity L2.  All normalized to values at the globule surface. 

H. Distribution of globules around the star as a function of position angle and projected radius. Note that they are concentrated in a broad equatorial torus.

**Comment by will-henney** [comment]
En la siguiente figura, hay que indicar el ángulo i.  Es el ángulo con el plano del cielo

![CleanShot 2023-06-18 at 15 48 42](https://github.com/will-henney/globule-seminario/assets/161264/4f9659ab-7a3b-4024-8056-2c3fcc8c5f03)

I. Correction for inclination angle (i).  

J. Empirically derived shell pressures as a function of projected radius (blue symbols).  The colored lines show the predicted pressure, assuming mechanical equilibrium with the free-flowing wind from the central star, for different inclination angles. 

K. Comparison of observed distribution of projected radii (orange) with distribution of true radii after deprojection (blue).

**Comment by will-henney** [comment]
![CleanShot 2023-06-18 at 15 58 08](https://github.com/will-henney/globule-seminario/assets/161264/fdee0dbd-75d1-42d7-bd68-4f960f992997)

From our analysis we find that the observed properties of the globules and interaction arcs are consistent with the two-wind interaction model if the inner globules are distributed in a shell with an inner radius of about 0.5 pc.  

Some of the globules in the outer part of the nebula show interaction arcs that are over-pressured with respect to the predicted ram pressure of the free-flowing stellar wind at that position.  We suggest that they are instead interacting with a modified wind that has already interacted with the inner globules. Mass loading from the turbulent wakes raises the pressure in the wind, as has been shown for the similar case of the Orion proplyds (García-Arredondo, Arthur, & Henney 2002).

**Comment by RobeReyes** [comment]
in Results I want write:

once we distinguished our globules in **A** and **B**, we fitted two Gaussians to know their surface brightness  (**E**) and obtain the density of the shell, which is about 1400 $cm^{-3}$, and finally we compared the pressure of the globules  $$P_{globule}=\rho c_s^2$$ and the RAM pressure caused by WR-124 $$P_{RAM}=\frac{\dot{M}v_\infty}{4\pi R^2}$$ also, we must correct for the inclination angle that may exist like in **I**, so we have $$R_{projected}=R\cos(i)$$ $$P_{RAM}(i)=P_{RAM}(0)cos^{5/2}(i)$$
with this we have the real distribution in radii in **K**.

**Reply by will-henney** [reply]
OK, except we are not measuring the properties of the globules (yet).  We are using only the measured properties of the shell. So I think that the first equation should be for P_shell and it should be with rho_2.

**Comment by RobeReyes** [comment]
The final poster
[poster final.pdf](https://github.com/will-henney/globule-seminario/files/11782819/poster.final.pdf)

**Reply by will-henney** [reply]
Perfect!
